hummingbird/forms.py

```diff
@@ -0,0 +1,25 @@
+from django import forms
+from models import UserProfile, UserDevice
+from django.contrib.auth.models import User
+
+class UserProfileForm(forms.ModelForm):
+	class Meta:
+		model = UserProfile
+		fields = ('name','song','length')
```

Comment:
nit: (optional) but end tuples with a comma. This is actually important for tuples with a single element, but you already have that covered here.